internal/store/engine_state.go

```diff
@@ -613,6 +613,16 @@ func resourceInfoView(mt *ManifestTarget) view.ResourceInfoView {
 			PodLog:             pod.CurrentLog,
 			YAML:               mt.Manifest.K8sTarget().YAML,
 		}
+	} else if mt.Manifest.IsLocal() {
```

Comment:
These doesn't even need to be `else`s since they return. If you reach the end of the function without returning then we can `panic`.